configs/12.0/packages/mpfr/sources

```diff
@@ -21,14 +21,14 @@
 
 ATSRC_PACKAGE_NAME="GNU MPFR Library"
 ATSRC_PACKAGE_VER=4.0.1
-ATSRC_PACKAGE_REV=12864
+ATSRC_PACKAGE_REV=12909
```

Comment:
OK
```
$ svn cat --revision 12909 https://scm.gforge.inria.fr/anonscm/svn/mpfr/branches/4.0/VERSION
4.0.2-dev
```

There are more updates to this branch, but they could be fetch in a new commit.

Reply:
Indeed.  They're preparing to release version 4.0.2.
All the new updates appeared after I created this patch.

Reply:
Anyhow, I think this PR is good as is. (I hope I didn't give the impression that I don't).